pilot/pkg/kube/inject/testdata/webhook/TestWebhookInject_http_probe_rewrite_disabled_via_annotation_template.yaml

```diff
@@ -0,0 +1,19 @@
+initContainers:
```

Comment:
add `rewriteAppHTTPProbers: true`? Otherwise I suppose we generate the same patch because the helm flag is not enabled even before this PR.

Reply:
added thanks.